dotnet/src/IntegrationTests/Planning/SequentialPlanner/SequentialPlannerTests.cs

```diff
@@ -39,8 +39,8 @@ public async Task CreatePlanFunctionFlowAsync(bool useChatModel, string prompt,
         // Arrange
         bool useEmbeddings = false;
         IKernel kernel = this.InitializeKernel(useEmbeddings, useChatModel);
-        _ = kernel.ImportSkill(new EmailSkillFake());
-        TestHelpers.GetSkills(kernel, "FunSkill");
+        _ = kernel.ImportPlugin(new EmailSkillFake());
+        TestHelpers.GetPlugins(kernel, "FunSkill");
```

Comment:
These are weird... why are we calling this but not getting the results? If it's deliberate, can you update with 
`_ = TestHelpers.GetPlugins....`